src/components/ClauseComponent.js

```diff
@@ -191,19 +197,52 @@ function ClauseComponent(props) {
       { !props.readOnly
         && <>
           <S.TestWrapper {...iconWrapperProps}>
-            <S.ClauseIcon {...testIconProps}>
+            <S.ClauseIcon 
+              {...testIconProps}
+              onMouseOver={() => setHoveringTestIcon(true)}
+              onMouseOut={() => setHoveringTestIcon(false)}
```

Comment:
These should be in wrappers instead of the icons themselves. The issue is the mouse going over the explicit lines in the `svg` are toggling this, instead of the whole area of the icon itself.

Reply:
@irmerk Is the issue still there ? I think it's fixed in this commit please check once at https://deploy-preview-318--cicero-ui.netlify.com/

I have nested this along with the ```svg``` because the hover is on that as well. Therefore, for a coherent behavior that the tooltips are only seen when the cursor is over the icon and the hover effect takes place. I implemented this.
Previously, I used the ```IconWrapper``` only

Reply:
Yes, still there. I'm saying this should look like:

```js
          <S.EditWrapper 
              onMouseOver={() => setHoveringEditIcon(true)}
              onMouseOut={() => setHoveringEditIcon(false)}
               {...iconWrapperProps}
          >
            <S.ClauseIcon {...editIconProps}>
```

for all the icons.

Reply:
@irmerk I have made these changes. Please review!